offline/cd.sh

```diff
@@ -4,12 +4,11 @@ set -euo pipefail
 
 function cleanup {
   (cd terraform/examples/wire-server-deploy-offline-hetzner ; terraform destroy -auto-approve)
-  echo done
 }
 trap cleanup EXIT
-
 (cd terraform/examples/wire-server-deploy-offline-hetzner ; terraform init ; terraform apply -auto-approve )
 adminhost=$(cd terraform/examples/wire-server-deploy-offline-hetzner ; terraform output adminhost)
+adminhost="${adminhost//\"/}"
```

Comment:
Why is this needed? Has the output format of `terraform output` changed between versions (`0.13` -> `1.x`)?